examples/imagenet/train_test.py

```diff
@@ -55,16 +55,16 @@ def test_create_model_local(self):
 
     Uses smaller inputs than `test_create_model` to due to higher compute.
     """
-    model = train.create_model(model_cls=models._ResNet1Local, half_precision=False)  # pylint: disable=protected-access
+    model = train.create_model(
+        model_cls=models._ResNet1Local, half_precision=False
+    )  # pylint: disable=protected-access
```

Comment:
This slightly changed the line for the `pylint` flag.

Reply:
yeah, this is one annoying change that ideally we'd have pyink/black fix upstream....